pandas/io/json/_normalize.py

```diff
@@ -3,13 +3,14 @@
 
 from collections import defaultdict
 import copy
-from typing import DefaultDict, Dict, List, Optional, Union
+from typing import Any, DefaultDict, Dict, Iterable, List, Optional, Union
 
 import numpy as np
 
 from pandas._libs.writers import convert_json_to_lines
 
-from pandas import DataFrame
+from pandas import DataFrame, isnull
```

Comment:
Would rather `import pandas as pd` as less prone to circular imports

Reply:
Okay. I'm assuming that you don't expect a refactor for this only for the function.